code/datums/spells/lightning.dm

```diff
@@ -0,0 +1,89 @@
+/obj/effect/proc_holder/spell/targeted/lightning
+	name = "Lightning Bolt"
+	desc = "Throws a lightning bolt at the nearby enemy. Classic."
+	charge_type = "recharge"
+	charge_max	= 300
+	clothes_req = 1
+	invocation = "UN'LTD P'WAH!"
+	invocation_type = "shout"
+	range = 7
+	cooldown_min = 30
```

Comment:
30 seconds at x1
23.3 seconds at x2
16.5 seconds at x3
9.8 seconds at x4
3 seconds at x5

Is it safe to cast this when the cooldown is shorter than the max charge time?
Do you want it to ramp that efficiently?